keyboards/handwired/dactyl_manuform/5x6/keymaps/via/keymap.c

```diff
@@ -0,0 +1,44 @@
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
```

Comment:
```suggestion
enum layers {
    _QWERTY,
    _LOWER,
    _RAISE,
};
```

Reply:
Thanks! Want me to also update this in the default layout I copied this from?

Reply:
> Thanks! Want me to also update this in the default layout I copied this from?

Would be best not to, as it would mean this PR would have to change it's target branch to `develop` 😵

If it's of interest, i have a [dactyl_manuform revamp](https://github.com/dunk2k/qmk_firmware/tree/dac_man_revamp/keyboards/handwired/dactyl_manuform) lurking when PRs of mine are merged; which includes `via` keymaps for all dactyl manuform matrix sizes.

Reply:
Oh that's great! Absolutely of interest.